src/main/java/com/google/api/codegen/discogapic/transformer/DiscoGapicNamer.java

```diff
@@ -182,13 +182,14 @@ public static Name getResponseName(Method method) {
 
   /**
    * Get the canonical path for a method, in the form "(%s/\{%s\})+" e.g. for a method path
-   * "{project}/region/{region}/addresses", this returns "project/{project}/region/{region}".
+   * "{project}/regions/{region}/addresses", this returns "projects/{project}/regions/{region}".
```

Comment:
you still need `projects/` at the beginning of the first example.

Reply:
So the first example is an example of a non-normalized path, the input to the function. Specifically, it's verbatim the path for compute.addresses.list as given in the Discovery doc. 

I wanted to not have the "projects/" prefix in the first example in order to contrast it with the canonical form, which does have the "projects/" prefix.

Reply:
derp I wasn't paying close enough attention...